examples/SharedMemory/SharedMemoryPublic.h

```diff
@@ -53,6 +53,7 @@ enum EnumSharedMemoryClientCommand
 	CMD_REQUEST_BODY_INFO,
 	CMD_REQUEST_INTERNAL_DATA,
 	CMD_STEP_FORWARD_SIMULATION,
+    CMD_SET_SPARSE_SDF,
```

Comment:
best to add those at the end, to avoid version incompatibilities. If not at the end, of changing existing structures, the version at the top of SharedMemoryPublic.h  needs to be updated.